mlir/include/mlir/Tools/mlir-opt/MlirOptMain.h

```diff
@@ -202,6 +218,11 @@ class MlirOptMainConfig {
   /// Configuration for the debugging hooks.
   tracing::DebugConfig debugConfig;
 
+  /// Verbosity level of diagnostic information. 0: Errors only,
+  /// 1: Errors and warnings, 2: Errors, warnings and remarks.
+  VerbosityLevel diagnosticVerbosityLevelFlag =
```

Comment:
The default should be ErrorsWarningsAndRemarks. I think some tests are breaking because of this :)